u_tls_extensions.go

```diff
@@ -766,6 +768,86 @@ func (e *ApplicationSettingsExtension) Write(b []byte) (int, error) {
 	return fullLen, nil
 }
 
+// ApplicationSettingsExtensionNew represents the TLS ALPS codepoint extension introduced by Chrome 133.
+// More information can be found here: https://chromestatus.com/feature/5149147365900288
+// TODO: This probably should be implemented differently
+type ApplicationSettingsExtensionNew struct {
```

Comment:
Assuming only the codepoint changed, maybe create an unexported version of `ApplicationSettingsExtension` with a codepoint variable and use it in `Read`. Then the two exported extensions can be embedded it, only changing the codepoint variable before calling `Read`. This way we don't have to keep two versions of the same extension.